src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -112,7 +114,156 @@ runnable example.
+        .UseOtlpExporter();
+    ```
+
+    For details see: [Meter](../../docs/metrics/customizing-the-sdk/README.md#meter).
+
+    When using `Microsoft.Extensions.Hosting` v8.0.0 or greater (a standard part
+    of ASP.NET Core) `Meter`s and `Instrument`s can also be enabled using
+    configuration.
+
+    `appSettings.json` metrics configuration example:
+
+    ```json
+    {
+      "Metrics": {
+        "EnabledMetrics": {
+          "Microsoft.AspNetCore.*": true,
+          "System.*": true,
+          "MyCompany.*": true,
+        }
+      }
+    }
+    ```
+
+    For details about the built-in metrics exposed by .NET see: [Built-in
+    metrics in
+    .NET](https://learn.microsoft.com/dotnet/core/diagnostics/built-in-metrics).
+
+  * Tracing
+
+    Trace telemetry is controlled by calling `TracerProviderBuilder.AddSource`
+    to listen to
+    [ActivitySource](https://learn.microsoft.com/dotnet/api/system.diagnostics.activitysource)s
+    emitting traces. Typically instrumentation packages will make this call
+    automatically.
+
+    Examples:
+
+    ```csharp
+    appBuilder.Services.AddOpenTelemetry()
+        .UseOtlpExporter()
+        .WithTracing(tracing => tracing
+            .AddSource(MyActivitySource.Name) // Listen to custom telemetry
+            .AddAspNetCoreInstrumentation() // Use instrumentation to listen to telemetry
+        );
+    ```
+
+    ```csharp
+    appBuilder.Services.ConfigureOpenTelemetryTracerProvider(tracing => tracing
+        .AddSource(MyActivitySource.Name) // Listen to custom telemetry
+        .AddAspNetCoreInstrumentation() // Use instrumentation to listen to telemetry
+    );
+
+    appBuilder.Services.AddOpenTelemetry()
+        .UseOtlpExporter();
+    ```
+
+    For details see: [Activity Source](../../docs/trace/customizing-the-sdk/README.md#activity-source).
+
+* The exporter registered by `UseOtlpExporter` will be added as the last
+  processor in the pipeline established for logging and tracing.
+
+* `UseOtlpExporter` can only be called once. Subsequent calls will result in a
+  `NotSupportedException` being thrown.
+
+* `UseOtlpExporter` cannot be called in addition to signal-specific
+  `AddOtlpExporter` methods. If `UseOtlpExporter` is called signal-specific
+  `AddOtlpExporter` calls will result in a `NotSupportedException` being thrown.
+
+### Configuring signals when using UseOtlpExporter
+
+`UseOtlpExporter` supports the full set of [environment
+variables](#environment-variables) listed below including the signal-specific
+overrides and users are encouraged to use this mechanism to configure their
+exporters.
+
+> [!NOTE]
+> In OpenTelemetry .NET environment variable keys are retrieved using
+  `IConfiguration` which means they may be set using other mechanisms such as
+  defined in appSettings.json or specified on the command-line.
+
+An `UseOtlpExporter` overload is provided which may be used to set the protocol
+and base endpoint but no other options are currently exposed:
```

Comment:
I think we should omit the part about only so much being currently exposed as it might come across as we are going to add more. We can always add more overloads in future if need be but it need not be mentioned like this.

```suggestion
and base endpoint:
```